src/components/ClauseComponent.js

```diff
@@ -66,7 +66,7 @@ function ClauseComponent(props) {
     };
 
     const findConditionals = node => ({
-      ...(((node.type === 'conditional') && (node.data.get('whenFalse') === '')) ? {
+      ...(((node.type === 'conditional')) ? {
```

Comment:
Sorry to keep going back and forth, but I think we want this check still in here. We don't want to keep conditionals that are this/that in the state here, only conditionals that are this/nothing. So it's a double check on this with L157 in this file

```js
value.isFalse && (value.whenFalse === '')
```

Reply:
No problem at all! I thought it was being redundant. Sorry!

Reply:
I suppose it is, but I'd rather have an extra check here and the peace of mind.

Reply:
Sure @irmerk this was my bad!